src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -11,7 +11,7 @@
  */
 public class AbsolutePath implements Path {
     public static final String INVALID_ABSOLUTE_PATH = "Invalid absolute path";
-    public static final String VALIDATION_REGEX = "\\/||(\\/\\p{Alnum}+)+\\/?";
+    public static final String VALIDATION_REGEX = "\\/|(\\/([a-zA-Z0-9]+\\s+)*[a-zA-Z0-9]+)*\\/?";
```

Comment:
@ljiazh3ng this first part `\\/` is actually unnecessary anymore.
The regex can be `(\\/(\\p{Alnum}+\\s+)*\\p{Alnum}+)*\\/?` only!